tests/cases/seniors_app/case_seniors_app_home.yaml

```diff
@@ -1128,4 +1128,343 @@ TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile:
       Role: desktopChrome
   # Go back to the iPad and fill in the Post Call Survey.
     - Type: case
-      Value: SeniorGreatPostSurveyCallHelper
\ No newline at end of file
+      Value: SeniorGreatPostSurveyCallHelper
+
+# CXTB-466: Verify "Emergency Services" button in portal
```

Comment:
You need to create a manual test case to link to the automation. I couldn't find the test case for this ticket

Reply:
Done and I saved it in test repository in never alone seniors app folden

Reply:
Need to add that test case number not the automation ticket number here